### PR TITLE
Added main benefit receiving variables and renamed jobseeker_support_…

### PR DESCRIPTION
**Pull request by verbman**
* Tax and benefit system evolution. Minor change.
* Impacted periods: all.
* Impacted areas: `openfisca_aotearoa/variables/acts/social_security/`
* Details:
* Added variables:
  - `social_security__receiving_main_benefit`
  - `emergency_benefit__receiving`
  - `sole_parent_support__receiving`
  - `supported_living_payment__receiving`
  - `young_parent_payment__receiving`
  - `youth_payment__receiving`
* Renamed variables:
  - `jobseeker_benefit__granted` to `jobseeker_support__granted`
- - - -

These changes:

- Impact the OpenFisca-Aotearoa public API (for instance renaming or removing a variable)
- Add a new feature (for instance adding a variable)

### CODE REVIEW

**Review by bonjourmauko — DISMISSED** [review]
Depends on #33 ;)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# Changelog
+# 17.0.0 - [37](https://github.com/govzeroaotearoa/openfisca-aotearoa/pull/37)
+* Added variables:
+  - `social_security__recieving_main_benefit`
+  - `emergency_benefit__receiving`
```
Comment (Br3nda) [inline CHANGELOG.md]: I before E, but not after C

```suggestion
  - `social_security__receiving_main_benefit`
  - `emergency_benefit__receiving`
```

File: openfisca_aotearoa/tests/social_security/jobseeker_support/jobseeker_support__base.yaml [hunk 2]
```diff
@@ -222,3 +222,9 @@
       2021-W23: [220.65, 0, 0]
       2022-W01: [102.04999, 0, 0]
       2023-W01: [251.5, 0, 0]
+    social_security__recieving_main_benefit:
```
Comment (Br3nda) [inline openfisca_aotearoa/tests/social_security/jobseeker_support/jobseeker_support__base.yaml]:
```suggestion
    social_security__receiving_main_benefit:
```

File: openfisca_aotearoa/tests/social_security/main_benefits.yaml [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        principal: ["Mama"]
+        partners: ["Papa"]
+  output:
+    social_security__granted_main_benefit:
+      2020-W01: [true, false]
+      2021-W01: [true, false]
+      2021-W23: [true, false]
+      2022-W01: [true, false]
+      2023-W01: [true, false]
+    social_security__recieving_main_benefit:
```
Comment (Br3nda) [inline openfisca_aotearoa/tests/social_security/main_benefits.yaml]:
```suggestion
    social_security__receiving_main_benefit:
```

**Review by Br3nda — APPROVED** [review]
looks good, but the spelling is wrong

**Review by Br3nda — DISMISSED** [review]
fix spelling

**Review by MarkCalvert — APPROVED** [review]

**Pull request merged**